vm/systemSmartContracts/esdt.go

```diff
@@ -979,6 +1080,10 @@ func (e *esdt) basicOwnershipChecks(args *vmcommon.ContractCallInput) (*ESDTData
 		e.eei.AddReturnMessage("not enough gas")
 		return nil, vmcommon.OutOfGas
 	}
+	if len(args.Arguments) < 1 {
```

Comment:
This change in basicOwnershipChecks method from 2 to 1 does not create some backwards compatibility issues

Reply:
no backward compatibility issues here.